src/app/modules/time-clock/components/entry-fields/entry-fields.component.ts

```diff
@@ -145,10 +157,20 @@ export class EntryFieldsComponent implements OnInit, OnDestroy {
       return;
     }
     this.entryForm.patchValue({ start_date: newHourEntered });
-    this.store.dispatch(new entryActions.UpdateCurrentOrLastEntry({ ...this.newData, ...this.entryForm.value }));
+    if (this.isTestUser) {
```

Comment:
This is not critical, but consider that this flag is not about if the user is tester or not. This flag is about if FT is enabled or not.

Reply:
Thanks for the comments, I will give the flag another name